### PR TITLE
feat: Add icon and option for Weblate in footer

### LINKED ISSUE #5635: Footer "Connect with us": Add icon and option for Weblate
Please add another option to add a link to Weblate to the Footer. 

The link we want to use is https://hosted.weblate.org/engage/open-event/, but it should be configurable just like the other options in this footer area.

Screenshot from 2020-11-15 11-17-07

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5635 

#### Short description of what this resolves:
require https://github.com/fossasia/open-event-server/pull/7451

add another option to add a link to Weblate to the Footer.
it is configurable just like the other options in this footer area.

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

__screenshots__

k1

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Please use monochrome svg icon

**Comment by sachinchauhan2889** [comment]
> Please use monochrome svg icon

@iamareebjamal sir, not able to find weblate monochrome svg icon. I think it is very rare icon. I tried on many sites but not get icon.  Thats why i used this.

**Comment by iamareebjamal** [comment]
Have you tried, ..., wait for it, ..., *drumrolls*, ...... https://hosted.weblate.org/?

**Comment by sachinchauhan2889** [comment]
wb

**Comment by sachinchauhan2889** [comment]
wbsa

File: app/templates/components/footer-main.hbs [hunk 1]
```diff
@@ -93,6 +93,11 @@
               <i class="telegram icon"></i> {{t 'Telegram'}}
             </a>
           {{/if}}
+          {{#if this.socialLinks.weblateUrl}}
+            <a class="item" href="{{this.socialLinks.weblateUrl}}" target="_blank" rel="noopener noreferrer">
+              <img src="{{if this.socialLinks.weblateUrl '/images/weblate icon.png'}}" class="weblate-icon"> {{t 'Weblate'}}
```
Comment (iamareebjamal) [inline app/templates/components/footer-main.hbs:98]: Monochrome SVG icon

**Comment by iamareebjamal** [comment]
> Please use monochrome svg icon

svg

**Comment by sachinchauhan2889** [comment]
https://semantic-ui.com/elements/icon.html
https://fontawesome.com/icons?d=gallery
https://fontawesome.com/icons?d=gallery&q=weblate

@iamareebjamal @mariobehling sir, there is no exact icon for weblate in font awesome and semantic ui icons.

**Comment by iamareebjamal** [comment]
> Have you tried, ..., wait for it, ..., _drumrolls_, ...... [hosted.weblate.org](https://hosted.weblate.org/)?

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**